bot/resources/utilities/stored_repos.json

```
@@ -0,0 +1,4 @@
+{
+    "kubernetes": "kubernetes/kubernetes",
+    "discord.py": "Rapptz/discord.py"
+}
```

```suggestion
    "peps": "python/peps",
}
```

It may also be worth adding python: python/cpython but this open to debate

---

are you sure we should map python to python/cpython or was that a typo? anyways i did add peps and cpython: python/cpython